### PR TITLE
[yoga] Bump to 1.19.0+20230215

### PR DESCRIPTION
**Pull request by xiaozhuai**
It has been years since yoga's last release. And there are many important fixes and improvement.
And yoga provide an official export, so we don't need provide an unofficial one.

### CODE REVIEW

File: ports/yoga/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "yoga",
   "version": "1.19.0",
-  "port-version": 1,
+  "port-version": 2,
```
Comment (dg0yt) [inline ports/yoga/vcpkg.json:4]: When not using the 1.19.0 release sources, then it needs a different version, such as `1.19.0+yyyymmdd` (with the date of the source revision).

Comment (xiaozhuai) [reply ports/yoga/vcpkg.json:4]: `1.19.0+20230215` or `1.19.0-20230215`?

Comment (xiaozhuai) [reply ports/yoga/vcpkg.json]: I change version to `1.19.0-20230215` and run `x-add-version` but why it remove previsous version `1.19.0` in `baseline.json`?

Comment (xiaozhuai) [reply ports/yoga/vcpkg.json]: Done

Comment (dg0yt) [reply ports/yoga/vcpkg.json]: The `+` is required. It indicates versions which should be regarded as newer than the plain version.
The `-` is for pre-release keywords (`-rc1). It marks version which must be regarded as older than the plain version.

https://github.com/microsoft/vcpkg-tool/blob/e9487f1afae094f5ee02d332a9b56b53e4425523/src/vcpkg/versions.cpp#L175-L216

Comment (xiaozhuai) [reply ports/yoga/vcpkg.json]: Done.
Thanks for pointing out this.

**Comment by MonicaLiu0311** [comment]
The usage test passed(header files were found). 
```
yoga provides CMake targets:

    find_package(yoga CONFIG REQUIRED)
    target_link_libraries(main PRIVATE yoga::yogacore)
```

**Review by MonicaLiu0311 — APPROVED** [review]

**Pull request merged**